Automattic/jetpack
# Blogroll: Append wpcom site to blogroll

Fixes #

## Proposed changes:


Adds feature to append WPCOM site to blogroll.

### Other information:


## Jetpack product discussion



## Does this pull request change what data or activity we track or use?



## Testing instructions:






* Checkout branch
* Sync to simple site / atomic site.
* Add blogroll block and open blogroll item appender popup.
* Verify you can filter through the suggestions and append to the blogroll block.
* Verify you can search for a WPCOM site using its URL and append it to the blogroll block.

<img width="964" alt="CleanShot 2023-09-14 at 00 36 23@2x" src="https://github.com/Automattic/jetpack/assets/10482592/a50ae3dc-a6fe-465d-a39e-53fe75592765">
